sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -125,6 +125,79 @@ def test_roc_curve_display_plotting(
     assert display.ax_.get_xlabel() == expected_xlabel
 
 
+@pytest.mark.parametrize("plot_chance_level", [True, False])
+@pytest.mark.parametrize(
+    "chance_level_kwargs",
+    [
+        {"linewidth": 1},
+        {"color": "b", "label": "DummyEstimator"},
+        {"color": "g", "linestyle": ":"},
+    ],
+)
+@pytest.mark.parametrize(
+    "constructor_name",
+    ["from_estimator", "from_predictions"],
```

Comment:
Can you put on a single line if it fits (remove the comma as I proposed in my previous suggestion).

Reply:
Sorry, I have made but not pushed these previous changes. I will push all of those together soon.